tests/functional/remote/07-slow-file-install/bin/my-rsync.sh

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env bash
+sleep 30
+exec rsync "$@"
```

Comment:
I can't see where in your test you are using this script. I tried commenting out the exec line, but the test still passes.

Reply:
Apologies I should have resolved this. Tim was referring to a commit where I had forgotten to copy across the global config so the test was broken.